sparse/client.go

```diff
@@ -9,26 +9,44 @@ import (
 	"io/ioutil"
 	"net/http"
 	"os"
+	"path/filepath"
 	"strconv"
 	"sync"
 	"time"
 
 	"github.com/pkg/errors"
 	log "github.com/sirupsen/logrus"
+
+	"github.com/longhorn/sparse-tools/types"
+	"github.com/longhorn/sparse-tools/util"
 )
 
 const (
-	httpClientTimeout = 5
-	numBlocksInBatch  = 32
+	httpClientTimeout      = 5
+	numBlocksInBatch       = 32
+	defaultSyncWorkerCount = 4
 )
 
+type DataSyncClient interface {
```

Comment:
The same, because it's used internally, so renaming methods to lower camel case intead.

Reply:
Addressed. Thank you.